src/pages/proposals/voting-result.js

```diff
@@ -80,15 +80,15 @@ class VotingResult extends React.Component {
 
     const quorum = parseBigNumber(currentRound.quorum, 0, false);
     const quota = parseBigNumber(currentRound.quota, 0, false);
-    const totalModeratorLockedDgds = parseBigNumber(daoInfo.totalModeratorLockedDgds, 0, false);
+    const totalLockedDgds = parseBigNumber(daoInfo.totalLockedDgds, 0, false);
```

Comment:
Note: This is just for `getProposalVotingPhaseStats`. `getModeratorVotingStats` (above) still uses `totalModeratorLockedDgds` for getting the quorum